CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Minikube Release Notes
 
+# Version 0.22.3 - 10/3/2017
+* Update dnsmasq to 1.14.5 [2022](https://github.com/kubernetes/minikube/pull/2022)
```

Comment:
I think this release will also includes the windows fixes

Reply:
Can you add the new PRs we just merged?  

* Components (apiserver, controller-manager, scheduler, kubelet) can now be configured in the kubeadm bootstrapper can now be configured with the --extra-config flag
* Kubeadm bootstrapper updated to work with Kubernetes v1.8.0
* OpenAPI registration fix cherry-picked for compatibility with kubectl v1.8.0

Reply:
done, tests have finished